### DISCUSSION TITLE
How run dashy on localhost only?

### DISCUSSION THREAD

**Discussion by PlusaN** [Lissy93/dashy#667] [Help and Support] [answered]
Hi!

I'm trying to deploy Dashy from source (bare metal server). I have set HOST and PORT environmental variables .
Port was changed, but HOST variable was ignored and Dashy service was started on *:PORT.
How run dashy on localhost only?

to start the server i use the command: `yarn start`

**Comment by lissy93** [comment]
[Here's instructions](https://github.com/Lissy93/dashy/blob/master/docs/deployment.md#build-from-source)

I'm not sure what you mean by localhost only, as this is what should happen anyway, you don't need to set the HOST variable.

**Reply by PlusaN** [reply]
I would like to change the service listen address to 127.0.0.1
![Screenshot-min](https://user-images.githubusercontent.com/61884717/169521217-c644d3de-1a4e-4a4e-8f64-2f0ff1dc3a8a.png)
For external access, I will install NGINX

**Comment by PlusaN** [comment, marked as answer]
It's worked for me: https://github.com/Lissy93/dashy/pull/671

**Reply by lissy93** [reply]
Nice, thanks for submitting it as a PR 🙌 
I'm just testing it out now, but it looks good to me...
